tests/_fixtures/mesos/overlay-master.json

```diff
@@ -7,47 +7,47 @@
           "backend": {
             "vxlan": {
               "vni": 1024,
-              "vtep_ip": "198.15.0.1\/20",
+              "vtep_ip": "198.15.0.1/20",
```

Comment:
Hmm... I don't remember why I did this actually... let me investigate, perhaps I can revert it.